prefab/prefab-publishing/mylibrary/build.gradle

```diff
@@ -39,14 +37,26 @@ android {
         }
     }
 
+    // Enable prefab intergrated inside AGP 4.1+
     buildFeatures {
         prefabPublishing true
     }
 
+    // Configure prefab for module headers
```

Comment:
Include the "mylibrary" module from the native build system in the AAR, and export the headers in src/main/cpp/include to its consumers.

Reply:
updated